doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+- The Shapley values are fairly distributed among the feature values
+- Shapley values can be easily interpreted and visualized
+
+**Cons**
+- Only applies to Tree-based models
+- Uses an approximation of the interventional conditional expectation instead of computing it directly
+
+#### Interventional Tree SHAP
+
+The interventional TreeSHAP method takes a different approach. Suppose we sample a reference data point, $r$, from the training dataset. Let $F$ be the set of all features. For each feature, $i$, we then enumerate over all subsets of $S\subset F \setminus \{i\}$. If a subset is missing a feature, we replace it with the corresponding one in the reference sample. We can then compute $f(S)$ directly for each coalition $S$ to get the Shapley values. One major difference here is combining each $S$ and $r$ to generate a data point. Enforcing independence of the $S$ and $F\setminus S$ in this way is known as intervening in the underlying data distribution and is where the algorithm's name comes from. Note that this breaks any independence between features in the dataset, which means the data points we're sampling won't be realistic.
+
+For a single Tree and sample $r$ if we iterate over all the subsets of $S \subset F \setminus \{i\}$, the interventional TreeSHAP method runs with $O(M2^M)$. Note that there are two paths through the tree of particular interest. The first is the instance path for $x$, and the second is the sampled/reference path for $r$. Computing the Shapley value estimate for the sampled $r$ will involve replacing $x$ with values of $r$ and generating a set of perturbed paths. Instead of iterating over the sets, we sum over the paths. Doing so is faster as many of the routes within the tree have overlapping components. We can compute them all at the same time instead of one by one. Doing this means the Interventional TreeSHAP algorithm obtains $O(LD)$ time complexity.
+
+Applied to a random forest with $T$ trees and using $R$ samples to compute the estimates, we obtain $O(TRLD)$ time complexity. The fact that we can sum over each tree in the random forest results from the linearity property of Shapley values.
+
+**Pros**
+- Very fast for a valuable category of models
+- The Shapley values are fairly distributed among the feature values
+- Shapley values can be easily interpreted and visualized
+- Computes the interventional conditional expectation exactly unlike the path-dependent method
+
+**Cons**
+- Less general as a white box method
+- Requires access to the dataset
+- Typically, slower than the path-dependent method
+
+### Global Feature Attribution
+
+Global Feature Attribution methods aim to show the dependency of model output on a subset of the input features. They are a global insight as it describes the behavior of the model over the entire input space. For instance, Accumulated Local Effects plots obtain graphs that directly visualize the relationship between feature and prediction.
```

Comment:
2nd sentence grammar, referring to the same thing as "They" and "it". I would just go with something like "They provide global insight describing the..."

Reply:
Last sentence worth emphasizing globality again, something like "... between feature and prediction over an  /entire dataset/entire sample of a dataset/specified dataset/specific set of samples/".